backend/api/Areas/Tools/Controllers/ImportController.cs

```diff
@@ -1,19 +1,20 @@
 using System.Linq;
 using AutoMapper;
-using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
 using Pims.Api.Areas.Tools.Helpers;
 using Pims.Api.Areas.Tools.Models;
 using Pims.Api.Models;
+using Pims.Api.Policies;
+using Pims.Dal.Security;
 using Pims.Dal.Services.Admin;
 
 namespace Pims.Api.Areas.Admin.Controllers
 {
     /// <summary>
     /// ImportController class, provides endpoints for managing parcels.
     /// </summary>
-    [Authorize(Roles = "system-administrator")]
+    [HasPermission(Permissions.SystemAdmin)]
```

Comment:
I can see you are cleaning up things (great!), but please keep changes related to the feature being implemented so things are easier to track if for some reason we need to revert a merge. I think changes like this would be better suited to a tech-debt PR.

Reply:
I generally agree, but know I'll forget to update this at a later point.  I'd suggest if we want to be keep things separate than I'd at the least need to add a `// TODO: tech-debt` statement so that I can find it again.